firefox-ios/Client/Application/SceneDelegate.swift

```diff
@@ -42,8 +42,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             Experiments.shared.initializeTooling(url: url)
         }
 
-        routeBuilder.configure(isPrivate: UserDefaults.standard.bool(forKey: PrefsKeys.LastSessionWasPrivate),
```

Comment:
do we want to remove `LastSessionWasPrivate` from the prefkeys as well?

Reply:
It is removed in this PR.